src/mango/test/01-index-crud-test.py

```diff
@@ -193,7 +193,7 @@ def test_recreate_index(self):
         pre_indexes = self.db.list_indexes()
         for i in range(5):
             ret = self.db.create_index(["bing"], name="idx_recreate")
-            assert ret is True
+            self.assertTrue(ret)
```

Comment:
`self.assertIs` is better. This assertion will be passed if ret would be not just `True`, but `"1"`, `42`, `["ololo"]`, e.g. any non-falsy value.